index.html

```diff
@@ -1,168 +1,139 @@
+                <div class="about-me-ctn">
+                    <h2>About me</h2>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Itaque, dolores, nesciunt possimus, numquam incidunt asperiores illum omnis exercitationem libero unde maiores soluta nihil quibusdam. Ratione vitae rerum rem numquam aspernatur.</p>
                 </div>
-
-                <div class="articel-item-2">
-                  <a href="#">LinkedIn</a>
+                <div class="about-me-ctn">
+                    <h2>Biography</h2>
+                    <table>
+                        <tbody>
+                            <tr>
+                                <td>01/2022 - bis heute</td>
+                                <td>Web Developement Student at neuefische GmbH</td>
+                            </tr>
+                            <tr>
+                                <td>02/2019 - 01/2022</td>
+                                <td>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Beatae, tempora?</td>
+                            </tr>
+                            <tr>
+                                <td>07/2018 - 02/2019</td>
+                                <td>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Maiores, quasi?</td>
+                            </tr>
+                            <tr>
+                                <td>08/2016 - 07/2018</td>
+                                <td>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nemo, expedita!</td>
+                            </tr>
+                        </tbody>
+                    </table>
                 </div>
-              </div>
-
-            </article>
-          </div>
-
-        </div>
-
-      </section>
-      <!-- about-me section - short intro about me -->
-      <section id="about-me">
-        <h2>About me</h2>
-        <div class="about-me-ctn">
-          <div class="about-me-item" >
-            <h3>About me</h3>
-            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis distinctio, perferendis earum consequatur fugiat ad quam tenetur modi eos. Ratione?</p>
-          </div>
-          <div class="about-me-item" id="biography">
-            <h3>Biography</h3>
-            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis distinctio, perferendis earum consequatur fugiat ad quam tenetur modi eos. Ratione?</p>
-          </div>
-        </div>
-      </section>
-    </main>
-    <!-- footer and social-media section e.g. linkedIn, instagram, facebook -->
-    <footer id="footer">
-      <nav class="nav-footer">
-        <a href="#skills" class="nav-footer-item">Skills</a>
-        <a href="#contact" class="nav-footer-item">Contact</a>
-        <a href="#recent-work" class="nav-footer-item">Recent Works</a>
-        <a href="#about-me" class="nav-footer-item">About me</a>
-      </nav>
-      <p>&copy;Copyright 2022</p>
-    </footer>
-    
-  </div>
```

Comment:
I like the way you organized your flex-objects with `ctn`and `item`. Makes it easy to me to follow!